src/api/util.rs

```diff
@@ -160,6 +160,8 @@ pub struct Packet<T: Pixel> {
   pub data: Vec<u8>,
   /// The reconstruction of the shown frame.
   pub rec: Option<Arc<Frame<T>>>,
+  /// The Reference Frame
+  pub ref_f: Option<Arc<Frame<T>>>,
```

Comment:
`input` or `source` might be a better name in the end.

To be discussed if `Option<(Arc<Frame<T>>, Arc<Frame<T>>)` wouldn't be more future proof even if it would have the problem of documenting what is what.

Reply:
Are you referring to putting rec and source in one element? I'm not sure that that would be cleaner/easier to reason about.